### ISSUE TITLE
Multiple To Addresses

### ISSUE THREAD

**Issue by Surender246** [sendgrid/sendgrid-nodejs#241]
When am sending mails using sendgrid-nodejs to multiple users in a single call by passing array in to field its getting delivered to all users but they are able to see the others email address in the to address which i dont want to show. What i need is send multiple emails but recipient should only able to view his email address but not others email addresses. Am new to sendgrid please help me in solving this.

Am using the following code in request and am able to send emails to all the users;

request.body = {
        "content": [
            {
                "type": "text/html",
                "value": "<html><p>Hello, world!</p><img src=[CID GOES HERE]></img></html>"
    }
  ],
        "from": {
            "email": "suren@example.com",
            "name": "Suren"
        },
        "personalizations": [
            {
                "headers": {
                    "X-Accept-Language": "en",
                    "X-Mailer": "MyApp"
                },
                "subject": "Hello, World!",
                "to": [
                    {
                        "email": "suren@example.com",
                        "name": "Suren"
        }
      ]
    }
  ],
        "reply_to": {
            "email": "suren@example.com",
            "name": "Suren"
        },
        "subject": "Hello, World!",
        "template_id": "am using my template id"
    };

    sg.API(request, function (response) {
        console.log(response.statusCode)
        console.log(response.body)
        console.log(response.headers)
    })

**Comment by mbernier** [comment]
tl;dr; The trick here would be to generate one personalizations for each recipient. 

Each personalization object is a distinct email. I like to think of the `to:` field in each personalization object the same way as the `to:` in an email in gmail. Everyone in that field will see "that" email. However, if you add personalizations for each recipient, they each get an email that may look the same, but they won't see people from other personalizations.
